### PR TITLE
Replace components on Flow Control Policies editor page

### PR DESCRIPTION
**Pull request by jiggsaw85**
**Jira task:** [https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?label=Screen_Conversion&selectedIssue=V5GA-60](https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?label=Screen_Conversion&selectedIssue=V5GA-60)

### CODE REVIEW

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
The `add` and `delete` buttons for sequence need to be converted to the new buttons as well

Screen Shot 2023-05-01 at 9 28 59
Screen Shot 2023-05-01 at 9 28 56

**Comment by Aviv-Galmidi** [comment]
Screen Shot 2023-05-01 at 10 32 04

Icon should be replaced with the new icon

**Comment by jiggsaw85** [comment]
@Aviv-Galmidi All changes are made. Test for this page had 24 errors, not simply to fix because, it was not only missing classes issue. I fixed 14 tests and skipped 10, but I can confirm that all works on the page as expected.

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

**Comment by Aviv-Galmidi** [comment]
Awaiting PR fixes

File: src/assets/styles/main.scss [hunk 1]
```diff
@@ -548,3 +548,11 @@ $color-light: hsl(0, 0%, 96%);
 .form-field {
   margin-bottom: 15px;
 }
+
+.my-1 {
```
[the remark this answers is not in the document]
Comment (SrdjanReblaze) [reply src/assets/styles/main.scss:552]: done

File: src/views/security/FlowControlPoliciesEditor.vue [hunk 2]
```diff
@@ -241,25 +252,26 @@
...
-                      <a class="is-size-7 has-text-grey-lighter add-button add-entry-button"
-                         data-qa="add-new-row-btn"
-                         title="add new row"
-                         tabindex="0"
-                         @click="setNewEntryIndex(sequenceIndex)"
-                         @keypress.space.prevent
-                         @keypress.space="setNewEntryIndex(sequenceIndex)"
-                         @keypress.enter="setNewEntryIndex(sequenceIndex)">
-                        <i class="fas fa-plus"></i></a>
+                    <td colspan="5" class="d-inline-flex">
```
[the remark this answers is not in the document]
Comment (SrdjanReblaze) [reply src/views/security/FlowControlPoliciesEditor.vue:255]: done

**Comment by SrdjanReblaze** [comment]
> Awaiting PR fixes

all done

**Pull request merged**